src/screens/SurveyOverviewScreen.tsx

```diff
@@ -58,11 +72,18 @@ const renderSectionHeader = ({
   return <SectionHeader title={title} />;
 };
 
-export const SurveyOverviewScreen = () => {
+export const SurveyOverviewScreen = ({ route }: Props) => {
   const { loading, refetch, surveySections } = useSurveySections();
   const RefreshControl = usePullToRefetch(refetch);
   const languages = useSurveyLanguages();
   const navigation = useNavigation<StackNavigationProp<any>>();
+  const { additionalProps } = route.params ?? {};
+
+  const { data } = useStaticContent({
+    name: additionalProps?.htmlName ?? '',
```

Comment:
Why `??` instead of `||`? this would result in `false` for example if `additionalProps?.htmlName` would be `false`.

<img width="100" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/2a2e8c08-d71d-406b-9db0-c04ff305d13d">

I guess we want a fallback in any falsey case here to an empty string, right?

Reply:
Yes, you are right. I didn't think of Skip for this place. I guess if we use skip, we don't need extra checks for name, right?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/d3e2bb5ccf4ca1da43e67f69877b7b68207df161